src/style.css

```diff
@@ -0,0 +1,160 @@
+/* #region block: cards */
+
+.body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  justify-content: center;
+
+  width: 198px;
+  height: 404px;
```

Comment:
dont hardcode exact hight size of card block
checklist:
![image](https://user-images.githubusercontent.com/89999954/185923033-2b325861-157e-4313-a9d1-eeff9c386ca8.png)